backend/src/main/java/gov/cdc/usds/simplereport/service/email/EmailProviderTemplate.java

```diff
@@ -0,0 +1,24 @@
+package gov.cdc.usds.simplereport.service.email;
+
+public enum EmailProviderTemplate {
+  /* These values map to SendGrid Dynamic Template IDs
+   *
+   * The templates are added and edited at: https://mc.sendgrid.com/dynamic-templates
+   * The value should match (but doesn't have to) the `name` of the SendGrid Dynamic Template
+   * and the template GUID *must* match the `Template ID` from the link above for the
+   * Dynamic Template.
+   */
+
+  // new account request next-steps email (to account requester)
+  ACCOUNT_REQUEST("d-7b0c7810ce5643f295e169f38b8db015");
```

Comment:
Hmmmm, that's a good point.  It isn't sensitive and also isn't likely to change (it would only change if the template was replaced, new versions do not change the guid).

It is also weird to bury a vendor-specific ID this deep.  I agree it is a good idea to move it into `application.yaml`.